src/HttpCache/EventListener/AddTagsListener.php

```diff
@@ -31,10 +31,15 @@
 final class AddTagsListener
 {
     private $iriConverter;
+    /**
+     * @var array
+     */
+    private $cacheResources;
 
-    public function __construct(IriConverterInterface $iriConverter)
+    public function __construct(IriConverterInterface $iriConverter, array $cacheResources)
```

Comment:
```suggestion
    public function __construct(IriConverterInterface $iriConverter, array $cacheResources = [])
```

(Backward compatibility)

Reply:
`$resources`, or `$cachedResources`?

Reply:
Actually resources which should be cached. But Ignore it, I'll try the annotation approach.